apps/challenges/views.py

```diff
@@ -168,6 +168,10 @@ def add_participant_team_to_challenge(request, challenge_pk, participant_team_pk
         response_data = {'error': 'Challenge does not exist'}
         return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
 
+    if challenge.end_date < timezone.now():
```

Comment:
Do add a check for future challenges.
`if challenge.end_date < timezone.now() or challenge.start_date > timezone.now():`
I think the condition will be ^ now. Please test it once.

Reply:
@RishabhJain2018 Sounds good! :D